### ISSUE TITLE
Provide convinience method for usages of the log in simple scripts

### ISSUE THREAD

**Issue by nemanja-boric-sociomantic** [sociomantic-tsunami/ocean#545]
Currently, to make the logger which will write inside the console for a simple application scripts, one needs to figure out how to configure loggers:

```D
import ocean.util.log.Logger;
import ocean.util.log.Appender;
import ocean.util.log.AppendStderrStdout;
import ocean.util.log.layout.LayoutSimple;

alias Appender.Layout Layout;

void main()
{
    Layout console_log_layout = new LayoutSimple;
    auto appender = new AppendStderrStdout(Logger.Level.Warn, console_log_layout);
    auto log = Log.lookup("myapp");
    log.add(appender);

    log.error("Hello world.");
}
```

We could/should provide a convenience method to grab a simple logger preconfigured to write to the console.

**Comment by gavin-norman-sociomantic** [comment]
>We could/should provide a convenience method to grab a simple logger preconfigured to write to the console.

How often is this required, though? We already have `Stdout`.

**Comment by nemanja-boric-sociomantic** [comment]
`Stdout` is not a true replacement for logging, though (what I find useful in one-script utilities is logging levels). I agree it's not critical, nor very important, but I think logging framework should support this use case easily.

**Comment by mihails-strasuns-sociomantic** [comment]
I think what you really want is https://github.com/sociomantic/ocean/issues/969

**Comment by nemanja-boric-sociomantic** [comment]
Yeah, I am ready to settle with anything that makes me not create layout, appender and configure the logger manually. Just imports alone are huge complexity for this.

**Comment by nemanja-boric-sociomantic** [comment]
The copy of the proposal in the issue:

To be able to use the logging subsystem ALWAYS for any debug output in libraries and programs, we should have an easy way to enable different loggers to output to the console. An easy way to do this, would be to add a `--debug=<logger>` flag to the `ocean.util.app.ext.LogExt` to enable a logger (probably at Trace level) to output to the console.

We can even use this format to be able to specify the level: `--debug=<logger>[:<level>]`, where the default level is Trace.

For this to be useful in certain context, it might make more sense to name loggers after some logic unit you want to debug, like SwarmClient. We could replace debug (Something) with this at some point (where it doesn't affect the performance, i.e. probably not in tight loops and things like that).

**Comment by Geod24** [comment]
Not tested, but I'd expect this to do what you want:

```D
import ocean.util.log.Logger;
import ocean.io.Console;

void main()
{
    Log.config(Cout.stream()); // This could be set a default argument, perhaps ?
    auto log = Log.lookup("myapp");
    log.error("Hello world.");
}
```
